env.list.sample

```diff
@@ -0,0 +1,15 @@
+AUGUR_DATABASE=
```

Comment:
put in the values from the readme

Reply:
Should I also simplify the instructions in the readme now that this file already exists? (i.e. remove the duplicate copy of the credentials and maybe just say "copy the example file and rename it to `.env` to use the default/public credentials"?

Reply:
yeah that sounds good!

Reply:
Done